keras/src/callbacks/orbax_checkpoint.py

```diff
@@ -0,0 +1,579 @@
+        initial_value_threshold: Floating point initial "best" value for the
+            monitor, used with `save_best_only`.
+        save_optimizer_state: Boolean, whether to include optimizer variables
+            in the checkpoint. Defaults to True.
+        save_on_background: Boolean, whether to save asynchronously in the
+            background. Defaults to True.
+        save_metadata: Dict or callable, additional metadata to save with each
+            checkpoint. If callable, it will be called with (epoch, logs) and
+            should return a dict. Defaults to None.
+        save_data_iterator: Dict or callable, data iterator state to save with
+            each checkpoint. If callable, it will be called with (epoch, logs)
+            and should return a dict with serializable iterator state.
+            Defaults to None.
+        save_metrics_state: Boolean, whether to include stateful metrics
+            variables in the checkpoint. Defaults to False.
+        async_timeout_secs: Integer, timeout in seconds for async checkpointing
+            operations. Defaults to 600 (10 minutes).
+        enable_background_delete: Boolean, whether to delete old checkpoints in
+            the background. Defaults to False.
+        post_finalization_callback: Callable, function to call after async
+            checkpointing operations complete. Defaults to None.
+        save_transforms: Dict of orbax.checkpoint.Transform objects to apply
+            during saving. Keys should match composite_state keys (e.g.,
+            'model_weights', 'optimizer_state'). Defaults to None.
+        save_decision_policy: orbax.checkpoint.SaveDecisionPolicy object to
+            control when checkpoints are saved. Currently supports
+            FixedIntervalPolicy for saving at regular intervals. If provided,
+            overrides the default save frequency logic. Defaults to None.
+        save_interval: Integer, save checkpoints every N steps. If provided,
+            overrides save_freq. Defaults to None.
+    """
+
+    def __init__(
+        self,
+        directory,
+        monitor="val_loss",
+        verbose=0,
+        save_best_only=False,
+        mode="auto",
+        save_freq="epoch",
+        max_to_keep=5,
+        keep_period=None,
+        initial_value_threshold=None,
+        save_optimizer_state=True,
+        save_on_background=True,
+        save_metadata=None,
+        save_data_iterator=None,
+        save_metrics_state=False,
+        async_timeout_secs=600,
+        enable_background_delete=False,
+        post_finalization_callback=None,
+        save_transforms=None,
+        save_decision_policy=None,
+        save_interval=None,
+    ):
+        if ocp is None:
+            raise ImportError(
+                "OrbaxCheckpoint requires the 'orbax-checkpoint' package. "
+                "Install it with: pip install orbax-checkpoint"
+            )
```

Comment:
You should get this error message for free with `LazyModule`.

Reply:
ok